lib/misc.js

```diff
@@ -113,8 +114,31 @@ function launchBrokerServer() {
         root: path.resolve(__dirname, '..'),
         cache: -1,    // disable caching,
         cors: true,
+        before: [
+          (req, res, next) => {
+            // Provide node_modules
+            let resolvedPath;
+            if (req.url.startsWith("/node_modules")) {
+              const moduleName = req.url.substr(14).replace('..', '');
```

Comment:
@kamijin-fanta The URLs which have a query string (ex: `/node_modules/mathjax/MathJax.js?config=TeX-MML-AM_CHTML-full,Safe`, `/node_modules/mathjax/config/TeX-MML-AM_CHTML-full.js?V=2.7.3`) couldn't be resolved because the query string isn't be excluded from `moduleName`. Could you fix this issue?